### PR TITLE
feat: Revamp (Local Storage, Event Linking, Layout, Error Highlights)

### LINKED ISSUE #17: Add stepper buttons that walk through updatevehicle logs
Update mark on map, update panorama if shown, update jsonview & log row highlight

### LINKED ISSUE #114: Add logging backend that can support downloaded log files
Cloud logging UI lets users select & then download logs.  We should support ingesting logs in that format.

### LINKED ISSUE #133: fleet-debugger should highlight rows in the logtable where the call failed
Right now it's not obvious w/o deeply inspecting the jsonpayload that a given operation actually failed.

### PR DESCRIPTION
**Pull request by regeter**
Fixes #17
Fixes #114
Fixes #133

I will be an active maintainer of this project going forward.
Revamp of Fleet-Debugger to be more focused on local storage importing of logs data.

Streamlined Log Export: Customers can easily export log files via Google Cloud Console's Cloud Logging UI.
Direct Log Upload: Users can upload log exports (in zip or JSON format) directly to Fleet Debugger.
Interactive Event Linking: Clicking an event simultaneously visualizes it on both the map and timeline for comprehensive analysis.
Responsive Design: Everything is client side. Keyboard shortcuts to rapidly step through the events.

Will update the manual/instructions in followup PR.

### CODE REVIEW

File: src/HighVelocityJump.js [hunk 1]
```diff
@@ -84,23 +87,42 @@ class HighVelocityJump {
...
-    return _(jumps)
-      .filter((e) => e.velocity >= computedOutlier)
-      .sortBy("velocity")
-      .value();
+    console.log("stdDevVelocity", stdDevVelocity);
+
+    // Consider a jump significant if:
+    // 1. Its velocity is greater than the median + 2 standard deviations
+    // 2. OR its velocity is greater than velocityOutlier (150 MPH)
+    // 3. AND the distance traveled is more than 1 meter
```
[the remark this answers is not in the document]
Comment (regeter) [reply src/HighVelocityJump.js:109]: Based on a dataset that created spurious jumps due to having 0 movement for the whole dataset which created a velocityOutlier of 0 and created jumps. This dataset is really an edge case that probably wouldn't happen in real world. Other values from 1m to ~25m would work just as well. This is a good reminder that we don't really have good test coverage, I will add that data into a test.
